website/blog/2018-05-29-jest-23-blazing-fast-delightful-testing.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Jest 23: 🚀 Blazing Fast Delightful Testing
+author: Rick Hanlon II
+authorURL: https://twitter.com/rickhanlonii
+authorFBID: 160800566
+---
+
+Today we are excited to announce Jest 23, our largest major release to date! Together with over 100 contributors, we've shipped a ton of features and bug fixes. Thank you to everyone in the community for helping make JavaScript Testing Delightful.
+
+We would also like to welcome Webpack to the Jest community! After converting from Mocha to Jest 23 Beta, their total test suite time was reduced 6x from over 13 minutes to 2 minutes 20 seconds. #blazingmeansgood
+
+Here's are some of the Jest 23 highlights and breaking changes. For a full list see the [changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md).
+
+## Interactive Snapshot Mode
+
+We've added a new default watch menu option we're calling Interactive Snapshot Mode. This new mode allows you to step through each failing snapshot in each failing suite to review your failed snapshots and choose to update or skip each individually.
+
+![Interactive snapshot mode in action](/jest/img/blog/23-interactive.gif)
+
+See the Interactive Snapshot Mode docs [here](https://facebook.github.io/jest/docs/en/snapshot-testing.html#interactive-snapshot-mode).
+
+## Snapshot Property Matchers
+
+Often times, objects you're snapshotting contain generated values like Dates and IDs. Jest now allows you to pass properties to the snapshot matcher which specify the structure of the data instead of the specific values. These property matchers are then verified before serializing the matcher type (instead of the value), giving you consistent snapshot results across test runs.
```

Comment:
not a native speaker, but 
![image](https://user-images.githubusercontent.com/1404810/40642061-30db7a1a-631c-11e8-807e-46240f35f733.png)

Reply:
:D